lib/opensusebasetest.pm

```diff
@@ -336,11 +333,6 @@ sub wait_boot {
             # harddisk' is above
             send_key_until_needlematch 'inst-bootmenu-boot-harddisk', 'up';
             boot_local_disk;
-            if (check_var('ARCH', 'aarch64') and get_var('UEFI')) {
```

Comment:
This has nothing with 'Fix ppc64le booting on zdup (poo#18228)', but I don't mind, leave it.